lib/rubocop/cop/rspec/example_without_description.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for examples without a description.
+      #
+      # RSpec allows for auto-generated example descriptions when there is no
+      # description provided or the description is an empty one.
+      #
+      # This cop removes empty descriptions.
+      # It also defines whether auto-generated description is allowed, based
+      # on the configured style.
+      #
+      # This cop can be configured using the `EnforcedStyle` option
+      #
+      # @example `EnforcedStyle: always_allow`
+      #   # bad
+      #   it('') { is_expected.to be_good }
+      #   it '' do
+      #     result = service.call
+      #     expect(result).to be(true)
+      #   end
+      #
+      #   it { is_expected.to be_good }
+      #   it do
+      #     result = service.call
+      #     expect(result).to be(true)
+      #   end
+      #
+      # @example `EnforcedStyle: single_line_only`
+      #   # bad
+      #   it('') { is_expected.to be_good }
+      #   it do
+      #     result = service.call
+      #     expect(result).to be(true)
+      #   end
+      #
+      #   good
+      #   it { is_expected.to be_good }
+      #
+      # @example `EnforcedStyle: disallow`
+      #   # bad
+      #   it { is_expected.to be_good }
+      #   it do
+      #     result = service.call
+      #     expect(result).to be(true)
+      #   end
+      class ExampleWithoutDescription < Cop
+        include ConfigurableEnforcedStyle
+
+        MSG_DEFAULT_ARGUMENT = 'Omit the argument when you want to ' \
+                               'have auto-generated description.'.freeze
+        MSG_ADD_DESCRIPTION  = 'Add a description.'.freeze
+
+        def_node_matcher :example?, Examples::ALL.send_pattern
+        def_node_matcher :example_description, '(send nil? _ $(str $_))'
+
+        def on_send(node)
+          return unless example?(node)
+
+          check_example_without_description(node)
+
+          example_description(node) do |message_node, message|
+            return unless message.to_s.empty?
```

Comment:
Just curious – what might `message` be besides a string?

Reply:
Nothing else, I just have the habit of converting to the expected type